contrib/harmonic_armature_guide.lua

```diff
@@ -0,0 +1,84 @@
+--[[
+  harmonic artmature guide for darktable
+
+  copyright (c) 2021  Hubert Kowalski
+  
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[
+HARMONIC ARMATURE GUIDE
+Harmonic Armature (also known as 14 line armature)
+
+INSTALLATION
+* copy this file in $CONFIGDIR/lua/ where CONFIGDIR is your darktable configuration directory
+* add the following line in the file $CONFIGDIR/luarc
+  require "harmonic_armature_guide"
+
+USAGE
+* when using guides, select "harmonic armature" as guide
+]]
+
+local dt = require "darktable"
+local du = require "lib/dtutils"
+local gettext = dt.gettext
+
+du.check_min_api_version("2.0.0", "harmonic_armature_guide") 
+
+-- Tell gettext where to find the .mo file translating messages for a particular domain
+gettext.bindtextdomain("harmonic_armature_guide",dt.configuration.config_dir.."/lua/locale/")
+
+local function _(msgid)
+  return gettext.dgettext("harmonic_armature_guide", msgid)
+end
+
+dt.guides.register_guide("harmonic armature",
+-- draw
+function(cairo, x, y, width, height, zoom_scale)
+  cairo:save()
+
+  cairo:translate(x, y)
+  cairo:scale(width, height)
+
+  cairo:move_to(0,0)
+  cairo:line_to(1, 0.5)
+  cairo:line_to(0.5, 1)
+  cairo:line_to(0,0)
+  cairo:line_to(1, 1)
+  cairo:line_to(0.5, 0)
+  cairo:line_to(0, 0.5)
+  cairo:line_to(1, 1)
+
+  cairo:move_to(1, 0)
+  cairo:line_to(0, 0.5)
+  cairo:line_to(0.5, 1)
+  cairo:line_to(1, 0)
+  cairo:line_to(0, 1)
+  cairo:line_to(0.5, 0)
+  cairo:line_to(1, 0.5)
+  cairo:line_to(0, 1)
+
+  -- middle lines - not needed for harmonic armature
+  -- cairo:move_to(0,0.5)
+  -- cairo:line_to(1,0.5)
+  -- cairo:move_to(0.5,0)
+  -- cairo:line_to(0.5,1)
+
+  cairo:restore()
+end,
+-- gui
+function()
+  return dt.new_widget("label"){label = _("harmonic armature"), halign = "start"}
```

Comment:
@johnny-bit in answer to your question

The widget you pass can be a box widget that's full of other widgets such as comboboxes that trigger callbacks when selected to change the lines that are displayed.

Reply:
Removed the lines that used anything that could be configureable. Maybe I'll go over it once this lands and i have more time... Or somebody else picks it up ;)